src/main/java/appeng/container/AEBaseContainer.java

```diff
@@ -117,6 +119,18 @@ public AEBaseContainer( final InventoryPlayer ip, final TileEntity myTile, final
 		this.prepareSync();
 	}
 
+	@Override
+	public void onCraftMatrixChanged( final IInventory par1IInventory )
+	{
+		this.onCraftMatrixChanged();
+	}
+
+	// TODO: this is the only thing the base class does, is this even needed?
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1135)